pkg/migrations/op_add_column.go

```diff
@@ -42,6 +42,12 @@ func (o *OpAddColumn) Start(ctx context.Context, conn db.DB, latestSchema string
 		}
 	}
 
+	if o.Column.Unique {
```

Comment:
In `addColumn` we already add `UNIQUE` to the generated SQL. You also need to change that function, so `pgroll` does not add unique constraint directly to the column. Similarly to what we do for check constraints.